package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "react_hw",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/client/app.js",
+  "dependencies": {
```

Comment:
Please check what dependencies could be moved into 'devDependencies' section and move them

Reply:
Pls check again